nginx/ngx_js_fetch.c

```diff
@@ -668,11 +668,20 @@ ngx_js_ext_fetch(njs_vm_t *vm, njs_value_t *args, njs_uint_t nargs,
 
 fail:
 
-    njs_vm_exception_get(vm, njs_value_arg(&lvalue));
+    ngx_log_debug2(NGX_LOG_DEBUG_EVENT, http->log, 0,
+                   "js http done fetch:%p rc:%i", fetch, NJS_ERROR);
```

Comment:
with %i specifier we need to cast NJS_ERROR to ngx_int_t.
Or, to use %d specifier.